internal/webhooks/inclusterippool.go

```diff
@@ -20,6 +20,13 @@ import (
 	"github.com/telekom/cluster-api-ipam-provider-in-cluster/pkg/types"
 )
 
+const (
+	// SkipValidateDeleteWebhookAnnotation is an annotation that can be applied
+	// to the InClusterIPPool or GlobalInClusterIPPool to skip delete
+	// validation. Necessary for clusterctl move to work as expected.
+	SkipValidateDeleteWebhookAnnotation = "ipam.cluster.x-k8s.io/skip-validate-delete-webhook"
```

Comment:
Can we import the package and use the constant that's defined here instead? https://github.com/pkbhowmick/cluster-api/blob/main/cmd/clusterctl/api/v1alpha3/annotations.go

Reply:
The delete-for-move annotation isn't in a released version of cluster-api yet, so this won't work for users using cluster-api v1.4.x. Unless the thinking is that a user would add the delete-for-move annotation manually, as was my original intention with this `skip-validate-delete-webhook` annotation.

Reply:
I just didn't want to redefine the constant. If it's not release on CAPI yet we can either wait, or if we need to get this in sooner we can keep the duplicated one, and maybe replace it later.

Reply:
Personally, I would prefer if we added this and replaced it later because at the moment clusterctl move doesn't work. However, I'm happy to go with whatever final decision you have if you have a strong opinion. I understand the desire to not rework this between versions too.

Reply:
Let's replace the constant once it's released (if we remember to 😄 )